docs/working-copy.md

```diff
@@ -80,3 +77,19 @@ See https://git-scm.com/docs/gitignore for details about the format.
 `.gitignore` files are supported in any directory in the working copy, as well
 as in `$HOME/.gitignore`. However, `$GIT_DIR/info/exclude` or equivalent way
 (maybe `.jj/gitignore`) of specifying per-clone ignores is not yet supported.
+
+
+## Workspaces
```

Comment:
Is this the equivalent of git worktrees?

Reply:
Yes